packages/front/src/routes/auth/reset-password/[token]/ResetPwd.controller.js

```diff
@@ -20,11 +20,7 @@ export class ResetPwdController {
         this.token = token;
         const urlParams = new URLSearchParams(window.location.search);
         this.activation = urlParams.get("active");
-        this.title = new Store(
-            this.activation ? "Activer mon compte en créant mon mot de passe" : "Modifier votre mot de passe",
-        );
-        this.validatitonTokenStore = new Store("waiting");
-        this.resetType = new Store(null);
+        (this.title = "Modifier votre mot de passe"), (this.validatitonTokenStore = new Store("waiting"));
```

Comment:
```suggestion
        (this.title = "Modifier votre mot de passe"), (this.validationTokenStore = new Store("waiting"));
```
typo nom de variable